src/Scan.js

```diff
@@ -339,12 +339,12 @@ class Scan extends React.Component {
     } = data;
     const {
       mutator: { checkIn },
-      stripes: { user },
       intl: { timeZone },
+      okapi,
     } = this.props;
     const { itemClaimedReturnedResolution } = this.state;
 
-    const servicePointId = get(user, 'user.curServicePoint.id', '');
+    const servicePointId = get(okapi, 'currentUser.curServicePoint.id', '');
```

Comment:
Valid fix, but the implication is that there's a bug in the ui-servicepoints event-handler (it needs to update two things but is only updating one). Alternatively, the bug is that `stripes.user` is supposed to be a reference to `okapi.currentUser`. Either way, we've definitely got a secondary problem to deal with here. Filed [UISP-37](https://issues.folio.org/browse/UISP-37)